docbrown/core/src/props.rs

```diff
@@ -1,166 +1,333 @@
-use crate::tprop_vec::TPropVec;
+use crate::lazy_vec::LazyVec;
```

Comment:
Is TPropVec used anywhere anymore?

Reply:
No, it was completely replaced with LazyVec